src/actions/vision.cpp

```diff
@@ -594,8 +594,8 @@ std::vector<int> get_sides_not_seeing(const unit & target)
 	const std::vector<team> & teams = resources::gameboard->teams();
 	std::vector<int> not_seeing;
 
-	std::size_t team_size = teams.size();
-	for ( std::size_t i = 0; i != team_size; ++i)
+	int team_size = static_cast<int>(teams.size());
+	for ( int i = 0; i != team_size; ++i)
 		if ( !target.is_visible_to_team(teams[i], false) )
 			// not_see contains side numbers; i is a team index, so add 1.
 			not_seeing.push_back(i+1);
```

Comment:
Not for this PR, but if we don't already have one, would a function for converting team numbers to side numbers and back be useful, instead of having obscure `+1` and `-1` (sometimes with a comment) around the code?

Reply:
I thought we already had that…? Should probably use it here. I think it'd look something like:

```cpp
std::size_t team_size = teams.size();
for ( std::size_t i = 1; i != team_size; ++i)
	if ( !target.is_visible_to_team(get_team(i), false) )
```

As a side note, I wouldn't mind if the formatting was fixed here while you're changing other things, though I also wouldn't mind if it's not.

Reply:
in this case it'd probably be simpler to do (not the scope of this pr know)
```
for ( const team& t: teams())
	if ( !target.is_visible_to_team(t, false) )
			not_seeing.push_back(t.side());
```

Reply:
Yeah, that looks good to me.